engines/dreamweb/stubs.cpp

```diff
@@ -2337,6 +2371,13 @@ void DreamWebEngine::obsThatDoThings() {
 	}
 }
 
+void DreamWebEngine::textToSpeech(const char *text) {
+	auto colon_pos = strchr(text, ':');
+	Common::String result(text, colon_pos ? colon_pos - text : strlen(text));
+	if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled_objects"))
+		_ttsMan->say(result);
+}
+
```

Comment:
Same remarks again:
 - Please do not use `auto`
 - Please move this inside the `if` block.